ocean/helium/data_transfer.py

```diff
@@ -8,14 +8,27 @@
 from boto3.s3.transfer import S3Transfer, TransferConfig, create_transfer_manager
 from s3transfer.subscribers import BaseSubscriber
 from tqdm.autonotebook import tqdm
+import xattr
 
 from .util import HeliumException, split_path
 
 
 HELIUM_METADATA = 'helium'
+HELIUM_XATTR = 'user.com.quiltdata.helium'
```

Comment:
On Linux, it looks like the user is only allowed to set `user.*` metadata. And I'm assuming that "com.quiltdata.*" is a good naming convention (at least, that's what Dropbox does).

Can someone check if it's the same in OS X? You may be able to use `getfattr` and `setfattr` commandline tools.

Reply:
I am able to set and remove on OS X with `xattr`; there is no `user.*` restriction:
```
apk-mbp-3:Desktop karve$ xattr test.txt 
com.quiltdata.helium
user.com.quiltdata.goo
user.com.quiltdata.helium
```

Reply:
Interesting. Can you see what Dropbox does (assuming you have it installed)? Every file inside the Dropbox folder should have xattrs.